java/kotlin-extractor2/src/main/kotlin/entities/Types.kt

```diff
@@ -9,8 +9,13 @@ private fun KotlinUsesExtractor.useClassType(
     c: KaClassType
 ): TypeResults {
     // TODO: this cast is unsafe; .symbol is actually a KaClassLikeSymbol
-    val javaResult = TypeResult(addClassLabel(c.symbol as KaClassSymbol) /* , TODO, TODO */)
-    val kotlinResult = TypeResult(fakeKotlinType() /* , "TODO", "TODO" */)
+    val classId = addClassLabel(c.symbol as KaClassSymbol)
+    val javaResult = TypeResult(classId /* , TODO, TODO */)
+    val kotlinTypeId =
+        tw.getLabelFor<DbKt_class_type>("@\"kt_class{$classId}\"") {
```

Comment:
Minor: We typically have a `;` separator between parts of the label. So I think this should be `"@\"kt_class;{$classId}\""`